inference-engine/src/transformations/src/transformations/convert_extract_image_patches_to_reorg_yolo.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/convert_extract_image_patches_to_reorg_yolo.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset3.hpp>
+#include <ngraph/rt_info.hpp>
+
+void ngraph::pass::ConvertExtractImagePatchesToReorgYolo::convert_extract_image_patches_to_reorg_yolo() {
+    auto image = std::make_shared<pattern::op::Label>(element::f32, Shape{1, 1, 1, 1});
+    auto extract_image_patches = std::make_shared<ngraph::opset3::ExtractImagePatches>(image, Shape{1, 1}, Strides{1, 1}, Shape{1, 1}, ngraph::op::PadType);
+
+    ngraph::graph_rewrite_callback callback = [](pattern::Matcher& m) {
+        auto extract_image_patches = std::dynamic_pointer_cast<ngraph::opset3::ExtractImagePatches>(m.get_match_root());
+
+        if (!extract_image_patches) {
+            return false;
+        }
+        
```

Comment:
cpp style check fails at this line:

convert_extract_image_patches_to_reorg_yolo.cpp:23:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]